src/wallet/rpcwallet.cpp

```diff
@@ -191,7 +191,11 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
     UniValue ret(UniValue::VOBJ);
     ret.pushKV("address", strAdd);
 
-    const CWDestination& dest = Standard::DecodeDestination(strAdd);
+    Standard::DecodeOptions options;
```

Comment:
in general in all rpcs wouldn't it be cleaner if you used directly 3 bool variables, instead of wrapping them in a `Standard::DecodeOptions options;`? Like:
`
bool isStaking, isShielded, isExchange = false;
`
so then you don't have to call everything with options.